### ISSUE TITLE
When should status switch from pending to running?

### ISSUE THREAD

**Issue by robnagler** [radiasoft/sirepo#6814]
Currently, runSimulation gets an immediate reply from the job_agent, before the job_cmd has even started. cancel_sim_test was written in such a way that it would catch this case when these semantics are changed. In #6784 pending will switch to running when job_cmd is ready to receive a fastcgi command. This seems reasonable to me, because nothing is running until job_cmd can load its imports and connect with the agent. 

I could have it reply immediately like it does now, but technically, it's in a queue (of one, see https://github.com/radiasoft/sirepo/issues/4154#issuecomment-1965461842) and isn't actually running.

@moellep opinions?

**Comment by moellep** [comment]
Yes, I think the closer we have the "running" state to the actual running sim, the better.

**Comment by robnagler** [comment]
[In a branch](https://github.com/radiasoft/sirepo/blob/839c8e6019c5eb2def368a6d77071e838ff291e6/sirepo/pkcli/job_agent.py#L615) there's a fix, but it is only for fastcgi. I think it's ok the way it is, because fastcgi will be the only mode for sequential sims going forward.
